react-ui/src/components/CreateAppointmentForm.js

```diff
@@ -0,0 +1,166 @@
+import React, {Component} from "react";
+import { Container, Row, Col } from 'react-grid-system';
+import { Redirect, Route, withRouter } from "react-router-dom";
+import Title from "./Title"
+import "./CreateAppointmentForm.css"
+import dashboard from "./Layouts/Dashboard"
+import { Link } from 'react-router-dom';
+
+import queryString from 'query-string'
+
+const axios = require('axios').default;
+
+//This class is for students to sign up for appointments 
+//students or support workers may access this page from the dashboard then fill in the following information 
+
+class CreateAppointmentForm extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        workerId: 8000000, // taken from Amy's test for the api
+        schoolId: 1,       //will need to implement a page before this to pass these values through
+        studentId: 0, // check with Melissa if already stored in props
+        workerTimeslotId : 0, 
+        purpose: "", // Max 300 => input size is 300
```

Comment:
Would the student be required to enter WorkerID in order to view the worker's available times? The student should be able to search by worker name, and would not know the worker's ID.

Reply:
No there will be a page before this where the student will select a worker they want to work with, then the WorkerID will be pass through.

Reply:
The workerID values should stay private and not be surfaced to the student for data privacy reasons. This is a good flow to ask the student to select the worker name (which they will know via the the worker description) and pass the workerID internally.